src/sourmash/tax/tax_utils.py

```diff
@@ -72,7 +73,7 @@ def collect_gather_csvs(cmdline_gather_input, *, from_file=None):
     return gather_csvs
 
 
-def load_gather_results(gather_csv, *, delimiter=',', essential_colnames=['query_name', 'name', 'f_unique_weighted', 'query_md5', 'query_filename'], seen_queries=set(), force=False):
+def load_gather_results(gather_csv, *, delimiter=',', essential_colnames=['query_name', 'name', 'f_unique_weighted', 'f_unique_to_query', 'unique_intersect_bp', 'remaining_bp', 'query_md5', 'query_filename'], seen_queries=set(), force=False):
```

Comment:
note that `seen_queries=set()` may not do what you're expecting here. tl;dr It's a quirk of Python to be wary of ;).

Briefly, since `load_gather_results` is defined on import of this file, the `set()` is evaluated once and assigned as the default value. If `seen_queries` is ever changed in the function without making a new copy, the default will _also_ be changed. Which can lead to really hard to debug bugs!

In this case I suggest making `seen_queries` into `None` and then in the function say `if seen_queries is None: seen_queries = set()`.

(The same is true of `essential_columns` being a list; you should probably change it to being a tuple, so that it's not modifiable.)

Reply:
thanks -- fixed, I think!